src/goto-instrument/cover.cpp

```diff
@@ -28,21 +28,27 @@ Date: May 2016
 void instrument_cover_goals(
   goto_programt &goto_program,
   const cover_instrumenterst &instrumenters,
-  message_handlert &message_handler)
+  message_handlert &message_handler,
+  const irep_idt mode)
```

Comment:
The valid values for `mode` are `ID_java` or anything else? Consider an enum if you don't want to use a `bool enable_java_blocks`

Reply:
`mode` is a standard concept used in the symbol table.